plugin/trino-druid/pom.xml

```diff
@@ -77,6 +77,11 @@
             </exclusions>
         </dependency>
 
+        <dependency>
```

Comment:
This dependency shouldn't' be required at all for this module, since the classes aren't referenced here.

Reply:
They are in the test (@Language annotation)